lib/cylc/cfgspec/globalcfg.py

```diff
@@ -529,6 +529,9 @@ def get_host_item(self, item, host=None, owner=None, replace_home=False,
                 if owner_home is None:
                     owner_home = os.path.expanduser('~%s' % owner)
                 value = value.replace(os.environ['HOME'], owner_home)
+        if item == "task communication method" and value == "default":
+            # Translate "default" to client-server comms: "https" or "http".
+            value = cfg['communication']['method']
```

Comment:
(In #2464 auth files are only copied over if comms method starts with "http" - but the value was "default"! - which refers to the general client/server comms method)

Reply:
I've check the codebase and I don't see any code that relies on the value being "default".

Reply:
I have just been bitten by this one while testing #2468.